packages/core/review-workflows/admin/src/index.ts

```diff
@@ -50,6 +52,19 @@ const admin: Plugin.Config.AdminInput = {
       });
     }
   },
+  bootstrap(app: StrapiApp) {
+    if (window.strapi.features.isEnabled(FEATURE_ID)) {
```

Comment:
would there be danger in injecting the component regardless of the flag? since we do the check in the header itself?

Reply:
Where do we do the check in the header? Maybe I missed that. I didn't have this but the components were still showing without the license. I saw review-workflows panel api and content-releases injection zone handle it this way.

Reply:
Oh this is a feature flag, not a future one, gotcha. Makes sense then